.go-version

```diff
@@ -1 +1 @@
-1.20.7
+1.22.2
```

Comment:
It needs checking separately, but I think we might be able to get rid of these files once we're consistently using the new go.mod toolchain directive.